pages/[username]/edit-profile.tsx

```diff
@@ -88,7 +109,7 @@ export default function EditBioPage(props: { userData: User }) {
             <div className="mt-8">
                 <div className="">
                     <h2 className="up-ui-title mb-4">
-                        Hide posts from global feed?
+                        Hide posts from public feeds?
```

Comment:
may be worth adding some extra explanation, like a little helper text underneath saying it will be hidden from global feed + profile